src/patternfly/components/Divider/divider.scss

```diff
@@ -67,6 +55,25 @@ $pf-c-divider--spacer-map: build-spacer-map("none", "xs", "sm", "md", "lg", "xl"
           --pf-c-divider--after--Inset: #{$spacer-value};
         }
       }
+
+      &.pf-m-vertical#{$breakpoint-name} {
+        display: inline-flex;
+        flex-direction: column;
+        width: auto;
+        height: inherit;
+        min-height: 100%;
+        max-height: 100%;
+
+        &::after {
+          flex-basis: calc(var(--pf-c-divider--m-vertical--after--FlexBasis) - var(--pf-c-divider--after--Inset));
+          width: var(--pf-c-divider--m-vertical--after--Width);
+        }
+      }
+
+      &.pf-m-horizontal#{$breakpoint-name} {
+        display: var(--pf-c-divider--Display);
+        flex-direction: row;
```

Comment:
While this works, I think we're going to need to reset all of the properties that the vertical modifier is changing, just so there are no issues with the way it behaves currently. If you look at the [Horizontal on lg example](https://patternfly-pr-4765.surge.sh/components/divider#horizontal-on-lg), the divider container is too tall because its `height` is set to `inherit`, since that came from `.pf-m-vertical`, and wasn't reset with `.pf-m-horizontal`.

Often what we do in this case is create `--base` vars for the default/base (horizontal) style properties, then assign those vars as the values for any default vars. We will also create placeholder vars for properties that the vertical variation changes, even if we didn't have to declare anything for that property initially in the default divider. We may do that for `height`, for example.

```
--pf-c-divider--Display--base: flex;
--pf-c-divider--Height--base: auto;
--pf-c-divider--Display: var(--pf-c-divider--Display--base);
--pf-c-divider--Height: var(--pf-c-divider--Height--base);
--pf-c-divider--m-horizontal--Display: inline-flex;
// --pf-c-divider--m-horizontal--Height: inherit; - you actually can't do this - `inherit` is a special word and will try to inherit `--pf-c-divider--m-horizontal--Height`, not evaluate to `inherit` where `var(--pf-c-divider--m-horizontal--Height)` is used

display: var(--pf-c-divider--Display); // you don't actually need this, just showing as an example. The line `@include pf-hidden-visible(var(--pf-c-divider--Display));` declares this for us.

&.pf-m-vertical[-on-{breakpoint}] {
  --pf-c-divider--Display: var(--pf-c-divider--m-vertical--Display);
  --pf-c-divider--Height: var(--pf-c-divider--m-vertical--Height, inherit); // means if `--pf-c-divider--m-vertical--Height` isn't defined, which it isn't, fall back to `inherit`. This allows the user to specify `--pf-c-divider--m-vertical--Height`, too, if they want.
}

&.pf-m-horizontal[-on-{breakpoint}] {
  --pf-c-divider--Display: var(--pf-c-divider--Display--base);
  --pf-c-divider--Height: var(--pf-c-divider--Height--base);
}
```

(This is unverified, so it may have errors but hopefully conveys the point!)

Reply:
Thank you for your feedback here @mcoker ! I've been investigating. Per your suggestion, I tried setting these base variables. It seems like the height of the container is coming from https://github.com/patternfly/patternfly/pull/4765/files#diff-7f3eddc914cec5fd7f6b03001bc43bd31c2ca2de891a5b6dc90e5565c0fe11b2R2 -- where `height` must be set to `3rem` in order for the vertical divider to render.

Does your suggested method still apply? Changing `--pf-c-divider--Height` like this doesn't seem to affect the height of `ws-example` once there's a switch from vertical to horizontal. Am I understanding correctly? What would you recommend?

Reply:
@jenny-s51 I think the height on the `.ws-preview` container is fine, we just don't want the horizontal divider's height to be set to `inherit` like it's set with a vertical divider. The `.ws-preview` having a fixed height with a horizontal divider is OK though. One thing that I think may help the example(s) that switch between vertical/horizontal using breakpoint modifiers would be to see if we can remove `display: flex` from the `.ws-preview` container in `Divider.css`. I did that locally and the vertical divider looked OK on my end, but I didn't test in other browsers, so you'd want to test that in FF, Safari, and Chrome if you give that a shot.